contributing.html

```diff
@@ -78,6 +78,20 @@ <h2>Becoming a Committer</h2>
 			<li>Demonstrated good understanding and exercised good technical judgement on at least one component of the codebase (e.g. core, clients, connect, streams, tests) from contribution activities in the above mentioned areas.</li>
 		</ul>
 
+		<h2>Collaborators</h2>
+
+		<p>
+		The Apache build infrastructure has provided two roles to make project management easier. These roles allow non-committers to perform some administrative actions like triaging issues or triggering builds. They are:
```

Comment:
I think it's meant to say triaging pull requests, right?

Reply:
```suggestion
		The Apache build infrastructure has provided two roles to make project management easier. These roles allow non-committers to perform some administrative actions like triaging pull requests or triggering builds. They are:
```

Reply:
Good point! I was just paraphrasing the ASF docs, but it would be confusing because we use Jira for issues, and this doesn't give you special Jira powers.